containers/Dockerfile.EESSI-build-node-debian10

```diff
@@ -25,3 +25,13 @@ RUN echo 'CVMFS_QUOTA_LIMIT=10000' > /etc/cvmfs/default.local \
 RUN mkdir -p /cvmfs/pilot.eessi-hpc.org
 
 RUN useradd -ms /bin/bash eessi
+
+# see https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html#cliv2-linux-install
+RUN curl https://awscli.amazonaws.com/awscli-exe-linux-$(uname -m)-2.2.35.zip -o awscliv2.zip \
```

Comment:
Should we avoid hard coding the version number?

Reply:
I actually prefer hardcoding it, to have control over what we install exactly...

But I'm happy to drop this, and always install latest instead.

Reply:
Ah, no, I ment we should store it in a variable or similar. We should certainly not just grab latest.